public/js/reader.js

```diff
@@ -447,6 +448,7 @@ Reader.toggleHeader = function () {
     localStorage.hideHeader = $("#i2").is(":visible");
     $("#toggle-header input").toggleClass("toggled");
     $("#i2").toggle();
+    Reader.applyContainerWidth();
```

Comment:
This seems to break double-page view: The loading spinner never hides even though the pages are fully loaded.

Reply:
Yeah sorry for the mess, that was fixed in pull #458 lol I discovered it afterwards but it would've caused a conflict with the preload pull, so I just added the fix there.

Reply:
Ah, that's on me for taking them out of order. I'll go check #458 and come back to this later I guess

Reply:
Saw the fix in the other PR so good to go here 👍